src/elbo.jl

```diff
@@ -1,6 +1,21 @@
 "Likelihoods which take a scalar (or vector of scalars) as input and return a single scalar."
 ScalarLikelihood = Union{BernoulliLikelihood,PoissonLikelihood,GaussianLikelihood}
 
+
+abstract type ExpectationMethod end
+struct Default <: ExpectationMethod end
+struct Analytic <: ExpectationMethod end
+
+struct GaussHermite <: ExpectationMethod
+    n_points
```

Comment:
```suggestion
    n_points::Int
```

Here you can enforce the type